docs/lua-records/functions.rst

```diff
@@ -200,7 +198,8 @@ Reverse DNS functions
   Returns a formatted hostname based on the format string passed.
 
   :param format: A hostname string to format, for example ``%1%.%2%.%3%.%4%.static.example.com``.
-  
+  :param exceptions: An optional table of overrides, for example ``{['10.10.10.10'] = 'quad10.example.com.'}``
```

Comment:
this does not make clear what these overrides do

Reply:
I added more words.